docs/detections/prebuilt-rules/rule-details/signed-proxy-execution-via-ms-workfolders.asciidoc

```diff
@@ -0,0 +1,121 @@
+[[signed-proxy-execution-via-ms-workfolders]]
+=== Signed Proxy Execution via MS WorkFolders
+
+Identifies the use of Windows Work Folders to execute a potentially masqueraded control.exe file in the current working directory. Misuse of Windows Work Folders could indicate malicious activity.
+
+*Rule type*: eql
+
+*Rule indices*:
+
+* winlogbeat-*
+* logs-windows.*
+
+*Severity*: medium
+
+*Risk score*: 47
+
+*Runs every*: 5 minutes
+
+*Searches indices from*: now-9m ({ref}/common-options.html#date-math[Date Math format], see also <<rule-schedule, `Additional look-back time`>>)
+
+*Maximum alerts per execution*: 100
+
+*References*:
+
+* https://docs.microsoft.com/en-us/windows-server/storage/work-folders/work-folders-overview
+* https://twitter.com/ElliotKillick/status/1449812843772227588
+* https://lolbas-project.github.io/lolbas/Binaries/WorkFolders/
+
+*Tags*:
+
+* Elastic
+* Host
+* Windows
+* Threat Detection
+* Defense Evasion
+
+*Version*: 1
+
+*Added ({stack} release)*: 8.2.0
+
+*Rule authors*: Elastic, Austin Songer
+
+*Rule license*: Elastic License v2
+
+==== Investigation guide
+
+
+[source,markdown]
+----------------------------------
+## Triage and analysis
+
+### Investigating Signed Proxy Execution via MS WorkFolders
+
+Work Folders is a role service for file servers running Windows Server that provides a consistent way for users to access
+their work files from their PCs and devices. This allows users to store work files and access them from anywhere. When
+called, Work Folders will automatically execute any Portable Executable (PE) named `control.exe` as an argument before
+accessing the synced share.
+
+Using Work Folders to execute a masqueraded control.exe could allow an adversary to bypass application controls and
+increase privileges.
+ 
+#### Possible investigation steps
+
+- Investigate the process tree starting with parent process WorkFolders.exe and child process control.exe to determine
+if other child processes spawned during execution.
+- Trace the activity related to the `control.exe` binary to identify any continuing intrusion activity on the host.
```

Comment:
For consistency, removing backticks for instances where we're not highlighting the name itself.
```suggestion
- Trace the activity related to the control.exe binary to identify any continuing intrusion activity on the host.
```